### DISCUSSION TITLE
IS_RENDER_TO_STREAM

### DISCUSSION THREAD

**Discussion by wibed** [StopNGo/react-proto#5] [General]
this variable is static and results to true always.

what are its effect?
what factors could change such variable?

i assume it is for some sort of backwards compatibility. 
arguing from there is it in respect to the browser?

**Comment by StopNGo** [comment]
Hello!
The files with constants are for the configuration. So, you can choose 'false' and it will be renderToString variant.

[serverRenderer.tsx](https://github.com/StopNGo/react-proto/blob/main/src/server/middlewares/serverRenderer.tsx):
```js
if (IS_RENDER_TO_STREAM) {
        ...
        const stream = renderToPipeableStream(jsx, {...})
} else {
        const reactHtml = renderToString(jsx)
        ...
        res.send(header + reactHtml + footer)
}
```
